demos/smart_classroom_demo/cpp/main.cpp

```diff
@@ -550,35 +553,38 @@ int main(int argc, char* argv[]) {
             return 1;
         }
 
-        slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
-        InferenceEngine::Core ie;
+        // slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
+        // InferenceEngine::Core ie;
+        slog::info << ov::get_openvino_version() << slog::endl;
+        ov::runtime::Core core;
 
         std::vector<std::string> devices = {FLAGS_d_act, FLAGS_d_fd, FLAGS_d_lm,
                                             FLAGS_d_reid};
         std::set<std::string> loadedDevices;
-
         for (const auto &device : devices) {
             if (loadedDevices.find(device) != loadedDevices.end())
                 continue;
-
             /** Load extensions for the CPU device **/
```

Comment:
models in this demo do not use extensions, so whole code block should be removed and README updated

Reply:
Done